components/playback/controls/PlateBarcode.vue

```diff
@@ -24,13 +24,14 @@
 <script>
 import { mapState } from "vuex";
 import playback_module from "@/store/modules/playback";
-
+import { TextValidation } from "@/js_utils/text_validation.js";
 /**
  * @vue-data {String} platebarcode - Current plate bar code
  * @vue-data {String} playback_state_enums - Current state of playback
  * @vue-computed {String} playback_state - Current value in Vuex store
  * @vue-event {String} validatePlateBarcode - User entered String parser
  */
+const TextValidation_plate_barcode = new TextValidation("platebarcode");
```

Comment:
Please have the string inside `TextValidation("platebarcode")` corrected also

Reply:
Agreed Completed.